src/components/10-atoms/radio/index.react.d.ts

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type Variant = 'foo' | 'bar';
+
+interface AXARadioProps {
+  variant?: Variant;
```

Comment:
here seems that lots of properties are missing

Reply:
I didn't plan to do typings.

Reply:
We always just copy pasta the props to typing as simple as possible. take example on the other components